r2dbc-mysql/build.gradle

```diff
@@ -0,0 +1,26 @@
+
+dependencies {
+    compile project(':mysql-async')
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    compile "io.r2dbc:r2dbc-spi:1.0.0.BUILD-SNAPSHOT"
+    implementation "io.reactivex.rxjava2:rxjava:2.2.7"
```

Comment:
Is it compatible or something else? Otherwise using `reactor-core` may be a better choice because of more powerful and concise API.

Reply:
I don't have a strong opinion about it, just followed what @mp911de did in this gist: https://gist.github.com/mp911de/9ea13939e8fd9a6b4ef138419f085715

Reply:
I used RxJava 2 in the vert.x example as vert.x had no dependency on Project Reactor. 
My choice for a Java 8 project would be Project Reactor, because it properly supports `CompletableFuture` (Java 8 types) and it has a `Context` that can be used to bind a connection to, if someone wants to properly manage a transaction.

Reply:
I changed it.